syncmaster/scheduler/celery.py

```diff
@@ -0,0 +1,6 @@
+# SPDX-FileCopyrightText: 2023-2024 MTS PJSC
+# SPDX-License-Identifier: Apache-2.0
+from syncmaster.scheduler import celery_factory
+from syncmaster.scheduler.settings import SchedulerAppSettings
+
+app = celery_factory(SchedulerAppSettings())
```

Comment:
Could we create celery app in `TransferJobManager.__init__`, and then patch this object in tests?

Reply:
The reason is that method `TransferJobManager.send_job_to_celery` should be static, therefore we can't pass celery app via self or as a parameter

Reply:
Added explanation